### PR TITLE
fix overflow, binding loop and cmake syntax

### PR DESCRIPTION
**Pull request by smk762**
- closes https://github.com/KomodoPlatform/atomicDEX-Desktop/issues/1852
- resolves `[18:09:35] [warning] [main.prerequisites.hpp:92] [3237681]: qrc:/Dex/Addressbook/AddAddressForm.qml:113:13: QML AddressTypeSelector: Binding loop detected for property "implicitHeight"`
- resolves quote syntax error in cmake post install

### CODE REVIEW

**Comment by cipig** [comment]
not sure if related to this, i usually don't use addressbook
anyway, if i try to add a new contact and get to this form
image
every time i try to change the coin like this
image
(selecting LTC from the search results)
i get this error on console
```
[16:08:27] [warning] [main.prerequisites.hpp:92] [2366618]: qrc:/Dex/Addressbook/AddressTypeSelector.qml:62:13: Unable to assign [undefined] to QString
```
and the coin is not changed

File: atomic_defi_design/Dex/Addressbook/AddAddressForm.qml [hunk 1]
```diff
@@ -113,8 +113,8 @@ Dex.Rectangle
             AddressTypeSelector
             {
                 id: addressTypeComboBox
-                Layout.fillWidth: true
-                Layout.fillHeight: true
+                width: 300
```
Comment (SylEze) [inline atomic_defi_design/Dex/Addressbook/AddAddressForm.qml:116]: Layout.preferredWidth

File: atomic_defi_design/Dex/Addressbook/AddAddressForm.qml [hunk 2]
```diff
@@ -113,8 +113,8 @@ Dex.Rectangle
...
+                height: 38
```
Comment (SylEze) [inline atomic_defi_design/Dex/Addressbook/AddAddressForm.qml:117]: Layout.preferredHeight

**Review by SylEze — CHANGES_REQUESTED** [review]

File: atomic_defi_design/Dex/Addressbook/AddAddressForm.qml (hunk 1 above)
Comment (smk762) [reply atomic_defi_design/Dex/Addressbook/AddAddressForm.qml:116]: using layout here results in `[19:03:30] [warning] [main.prerequisites.hpp:92] [4149538]: qrc:/Dex/Addressbook/AddAddressForm.qml:113:13: QML AddressTypeSelector: Binding loop detected for property "implicitHeight"`

File: atomic_defi_design/Dex/Addressbook/AddAddressForm.qml (hunk 2 above)
Comment (smk762) [reply atomic_defi_design/Dex/Addressbook/AddAddressForm.qml:117]: using layout here results in `[19:03:30] [warning] [main.prerequisites.hpp:92] [4149538]: qrc:/Dex/Addressbook/AddAddressForm.qml:113:13: QML AddressTypeSelector: Binding loop detected for property "implicitHeight"`

**Comment by cipig** [comment]
```
[23:07:49] [warning] [main.prerequisites.hpp:92] [2743785]: qrc:/Dex/Addressbook/AddAddressForm.qml:113:13: QML AddressTypeSelector: Binding loop detected for property "implicitHeight"
```

and when i try to change coin i get this
```
[23:08:09] [warning] [main.prerequisites.hpp:92] [2743785]: qrc:/Dex/Addressbook/AddressTypeSelector.qml:62:13: Unable to assign [undefined] to QString
```

**Review by SylEze — DISMISSED** [review]

**Review by Canialon — DISMISSED** [review]

**Comment by cipig** [comment]
getting this when i try to add a new address to an existing contact
image
in this case it's DGB, but DGB is already enabled

**Comment by smk762** [comment]
> getting this when i try to add a new address to an existing contact  in this case it's DGB, but DGB is already enabled

I get the same, only for individual coins (network standard is ok). @SylEze seems to be related to your most recent commits.
working: https://github.com/KomodoPlatform/atomicDEX-Desktop/actions/runs/2635869147
not working: https://github.com/KomodoPlatform/atomicDEX-Desktop/actions/runs/2649191429

**Comment by SylEze** [comment]
Fixed @cipig @smk762

**Review by SylEze — DISMISSED** [review]

**Review by SirSevenG — APPROVED** [review]
lgtm

**Comment by smk762** [comment]
One last issue @SylEze 

https://user-images.githubusercontent.com/35845239/178484180-3f955517-c1fe-4122-a6ea-06bd7d84139c.mp4

**Comment by smk762** [comment]
Thanks, looks good now :+1:

**Review by SylEze — APPROVED** [review]

**Review by SirSevenG — APPROVED** [review]

**Review by cipig — APPROVED** [review]

**Pull request merged**